src/Symfony/Component/HttpFoundation/Tests/RequestMatcher/SchemeRequestMatcherTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Tests\RequestMatcher;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestMatcher\SchemeRequestMatcher;
+
+class SchemeRequestMatcherTest extends TestCase
+{
+    public function test()
```

Comment:
We should have tests covering the fact of passing an array to the constructor, and also the fact of passing a comma-separated string (and even such a comma-separated string as one of the array items as we support it). Otherwise, the testsuite won't catch regressions on them.

Same is true for the method and ips matchers.